spec/lib/helpers/poa_access_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "helpers/poa_access"
+
+describe "WarRoom::PoaAccess" do
+  before { FeatureToggle.enable!(:poa_auto_refresh) }
+  after { FeatureToggle.disable!(:poa_auto_refresh) }
```

Comment:
🤔 I wonder if `FeatureToggle` is global state that is accessible to all tests running in parallel. If so, this could possibly cause flakiness with other tests that depend on it.

Maybe instead we could stub `FeatureToggle`, which will be isolated to our tests here?

```suggestion
  before do
    allow(FeatureToggle).to receive(:enabled?).
      with(:poa_auto_refresh, user: RequestStore.store[:current_user]) { true } 
  end
```